### DISCUSSION TITLE
Regressing out S.Score and G2M.Score

### DISCUSSION THREAD

**Discussion by rythi4** [satijalab/seurat#5726] [General]
I do not quite understand, when to regress out S. Score and G2M.Score. I have a dataset where I have tried to both regress out and not regress out S.Score and G2M.Score because I see some differences, when I split my PCA plot based on phase.

Is it wrong not to regress out S.Score and G2M.Score, if you remember to note that a certain cluster consists of proliferating cells? Or will it always be more correct to regress out these effects?

**Comment by nannabarnkob** [comment]
I am also quite interested in this subject. I think the major factor that determines whether it's "wrong" or not is to make sure you have an idea of which biological question you want to answer (not that that necessarily makes the decision any easier). Is it of interest that there may be proliferating cells in your dataset or does it tell you something about the cells..? 

An example of basing biological conclusions on cells that appear similar because they are proliferating is highlighted in this paper: 
Alquicira-Hernandez, J., Powell, J.E. and Phan, T.G. (2021), No evidence that plasmablasts transdifferentiate into developing neutrophils in severe COVID-19 disease. Clin Transl Immunol, 10: e1308. https://doi.org/10.1002/cti2.1308

Please let me know your thoughts cause I think it's quite difficult to decide as well.
